01-numpy.md

```diff
@@ -703,3 +703,22 @@ the graphs will actually be squeezed together more closely.)
 > ## Moving plots around {.challenge}
 >
 > Modify the program to display the three plots on top of one another instead of side by side.
+
+> ## Converting between an array and a vector {.challenge}
+>
+> In some cases in NumPy, an vector of size N behaves differently than an Nx1 array.
```

Comment:
Fair point. In my experience, it generally results in strange-looking broadcasting errors.

Perhaps I could reformat this as a small code fragment that results in such an error, and the challenge would be to fix the error. Do you think that would work better?